lib/http/options.rb

```diff
@@ -196,3 +196,4 @@ def argument_error!(message)
     end
   end
 end
+# rubocop:enable Metrics/ClassLength
```

Comment:
Maybe you can move the `disable` comment down to the line that opens the class and we won't need this? For instance:

```ruby
module HTTP
  class Options # rubocop:disable Metrics/ClassLength
```